src/common/components/clusterDetail/KubeconfigDownload.tsx

```diff
@@ -27,16 +26,8 @@ const KubeconfigDownload: React.FC<KubeconfigDownloadProps> = ({
     async (clusterId: Cluster['id'], status: Cluster['status']) => {
       const fileName = status === 'installed' ? 'kubeconfig' : 'kubeconfig-noingress';
       try {
-        if (ocmClient) {
-          const { data } = await ClustersAPI.getPresignedForClusterCredentials({
-            clusterId,
-            fileName,
-          });
-          saveAs(data.url);
-        } else {
-          const response = await ClustersAPI.downloadClusterCredentials(clusterId, fileName);
-          saveAs(response.data, fileName);
-        }
+        const response = await ClustersAPI.downloadClusterCredentials(clusterId, fileName);
```

Comment:
It is calling GET /v2/clusters/{cluster_id}/downloads/credentials

Reply:
this one was using prevously GET /v1/clusters/{cluster_id}/downloads/files-presigned
should be migrated to GET /v2/clusters/{cluster_id}/downloads/files-presigned